tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -3,7 +3,7 @@ source $SRCDIR/libtest.sh
 # Test DEVS= directive. Returns 0 on success and 1 on failure.
 test_devs() {
   local devs=$TEST_DEVS
-  local test_status
+  local test_status=1
```

Comment:
I think making test_status related cleanup is a good idea but you seem to be making only for 1 test while others test seem to have the same thing.

Whatever change you do, please keep it same for all tests.

I would suggest that remove test_status related change from this PR and generate a separate PR for all test_status related cleanup for all tests.